.evergreen/compile-unix.sh

```diff
@@ -1,6 +1,24 @@
-#!/bin/sh
+#!/bin/bash
 set -o errexit  # Exit the script with error if any of the commands fail
 
+# Find PHP binary path for the requested version
+if [ -z "$PHP_PATH" ]; then
+  if [ -d "/opt/php/${PHP_VERSION}-64bit/bin" ]; then
+     PHP_PATH="/opt/php/${PHP_VERSION}-64bit/bin"
```

Comment:
[build-linux-toolchain.sh](https://github.com/10gen/mongo-php-toolchain/blob/master/build-linux-toolchain.sh#L39) defaults to "64bit" but I see that "32bit" is also supported. Do any environments actually use the `FORCE_ARCH` parameter to opt into 32-bit? I traced this back as far as [.evergreen/config.yml](https://github.com/10gen/mongo-php-toolchain/blob/master/.evergreen/config.yml#L42), but can't tell where `${arch}` may come from.

Reply:
IIRC we no longer have any 32-bit systems that are supported in Evergreen, so this may be completely obsolete at this point.

Reply:
In that case, feel free to make a ticket to clean that up.